workbench-core/base/src/aws/helpers/s3Service.ts

```diff
@@ -75,4 +76,13 @@ export default class S3Service {
 
     await recursiveUpload(path, '');
   }
+
+  public async getPresignedUrl(s3BucketName: string, key: string, expirationMinutes: number) : Promise<string> {
+    // Sign the url
```

Comment:
Are you aware of the [Avoid S3 Presigned URLs or Keep Them Secure](https://aristotle.corp.amazon.com/implementations/407/) guidance in Aristotle? If not, please give it a look and ensure best practices are being followed. For instance, there should be verification here that the key being requested is a `uuid` (see the requirements section, item 5).

Reply:
Great! Thanks for looking into it.